product_variants_no_automatic_creation/views/product_view.xml

```diff
@@ -17,18 +17,54 @@
             <field name="model">product.template</field>
             <field name="inherit_id" ref="product.product_template_only_form_view" />
             <field name="arch" type="xml">
-                <button string="Variant Prices" position="attributes">
+            <xpath expr="//button[@name='103']" position="attributes">
```

Comment:
This number won't work in any database. You have to use the same expression as in original.